contrib/kind.sh

```diff
@@ -1163,5 +1167,11 @@ if [ "$KIND_INSTALL_PLUGINS" == true ]; then
 fi
 if [ "$KIND_INSTALL_KUBEVIRT" == true ]; then
   install_kubevirt
-  install_kubevirt_ipam_controller
+  deploy_primary_udn_nad
+  deploy_passt_binary
+
+  install_cert_manager
+  if [ ! "$KIND_OPT_OUT_KUBEVIRT_IPAM" == true ]; then
```

Comment:
worth to add please `KIND_OPT_OUT_KUBEVIRT_IPAM == true` to ipam-extensions repo, it will improve the flow there
no need to deploy HEAD and then cluster-sync the PR itself on PRs as done now
but just cluster-sync the PR, once this flag is used there
it improves stability and saves time

Reply:
answered [above](https://github.com/ovn-org/ovn-kubernetes/pull/4655/commits/bd1c2231f2bda37dc1f2e44b4d41c315aaf72f99#r1736279474).

Reply:
This is orthogonal of https://github.com/ovn-org/ovn-kubernetes/pull/4655#discussion_r1736279474
if thats what you mean, and can be done on a follow-up PR under ipam-ext
(the link you posted doesnt work)